sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_common.hpp

```diff
@@ -25,3 +25,20 @@
 
 // Map Level Zero runtime error code to UR error code.
 ur_result_t ze2urResult(ze_result_t ZeResult);
+
+// Controls Level Zero calls tracing.
+enum UrDebugLevel {
+  UR_L0_DEBUG_NONE = 0x0,
+  UR_L0_DEBUG_BASIC = 0x1,
+  UR_L0_DEBUG_VALIDATION = 0x2,
+  UR_L0_DEBUG_CALL_COUNT = 0x4,
+  UR_L0_DEBUG_ALL = -1
+};
+
+const int UrL0Debug = [] {
+  const char *DebugMode = std::getenv("UR_L0_DEBUG");
```

Comment:
@smaslov-intel : what do you think of this format? So instead of ZE_DEBUG, we would have UR_L0_DEBUG, and generally speaking, we would have UR_<ADAPTER>_<KEY>

Reply:
Looks reasonable to me (but remember not to remove existing SYCL_PI_LEVEL_ZER0_* vars yet)

Reply:
@smaslov-intel , right, I'm not removing anything. For now, only adding UR variables